<launch>
    <arg name="Joffsets" default="[40, 40, 5, 190, 75, 85]"/>

    <node name="psm_hand_ef" pkg="sitl_dvrk" type="psm_hand_ef.py" output="screen">
        <param name="arm_name"           type="str"       value="PSM2"/>
        <param name="expected_interval"  type="double"    value="0.001"/>
        <param name="calib_fn"           type="str"       value="/home/$(env USER)/aruco_data/psm2_calib_results_final_new_v2.mat"/>
        <param name="Tweight"            type="double"    value="0.8"/>
        <param name="Rweight"            type="double"    value="0.2"/>
        <rosparam param="Joffsets" subst_value="True"> $(arg Joffsets) </rosparam>
    </node>

    <node name="psm_hand_jaw" pkg="sitl_dvrk" type="psm_hand_jaw.py" output="screen">
        <param name="arm_name"           type="str"       value="PSM2"/>
        <param name="expected_interval"  type="double"    value="0.001"/>
        <param name="calib_fn"           type="str"       value="/home/$(env USER)/aruco_data/psm2_calib_results_final_new_v2.mat"/>
        <param name="Tweight"            type="double"    value="0.8"/>
        <param name="Rweight"            type="double"    value="0.2"/>
        <rosparam param="Joffsets" subst_value="True"> $(arg Joffsets) </rosparam>
    </node>

    <?ignore

    ?>
</launch>